oak/server/oak_manager.cc

```diff
@@ -55,6 +62,12 @@ void OakManager::CreateEnclave(const std::string& node_id, const std::string& mo
   ::oak::InitializeInput* initialize_input = config.MutableExtension(::oak::initialize_input);
   initialize_input->set_node_id(node_id);
   initialize_input->set_module(module);
+
+  // Uses a random port number in range of [node_port_min, node_port_max].
```

Comment:
Why even bother randomizing this? Could we just assign them sequentially? And for now I would say let us not even worry about keeping track of the free ones, so let's keep your TODO below.

Reply:
It is allocated sequentially in the latest commit